app/uk/gov/nationalarchives/omega/editorial/views/editSetRecordEdit.scala.html

```diff
@@ -218,6 +218,19 @@
         )
     },
     attributes = Map("rows" -> "5", "autofocus" -> "true")))
+@govukTextarea(Textarea(
+    id = "background",
+    name = "background",
+    value = editSetRecordForm.data.get("background"),
+    label = Label(forAttr = Some("background"), content = Text(messages("edit-set.record.edit.background"))),
```

Comment:
The `Textarea` needs a `classes` field set to the string value "govuk-label--s" to include this an an additional class, as shown in the prototype. You can read more about this at https://design-system.service.gov.uk/get-started/labels-legends-headings/

Reply:
<img width="1287" alt="image" src="https://user-images.githubusercontent.com/119881905/211327125-5be05604-5697-4095-85d1-c747264f727f.png">

Changed.